packages/react-native-web/src/exports/StatusBar/index.js

```diff
@@ -7,16 +7,92 @@
  * @providesModule StatusBar
  * @flow
  */
-
 import { Component } from 'react';
 
+
+const {head} = document
+
+let _barStyle = 'default'
+let _hidden = false
+let _translucent = false
+
+
+function getMetaTag(attrName) {
+  const htmlCollection = head.getElementsByTagName('meta')
+
+  let tag = Array.from(htmlCollection).filter(({name}) => name === attrName)[0]
+
+  if(!tag) {
+    tag = document.createElement('meta')
+    tag.name = attrName
+
+    head.appendChild(tag)
+  }
+
+  return tag
+}
+
+function setAppleMobileWebAppCapable() {
+  getMetaTag('apple-mobile-web-app-capable').content =
```

Comment:
Setting `apple-mobile-web-app-capable` is problematic (it's basically a broken feature), which is why most web apps do not set it.

Reply:
I've used it because seems [Apple docs say it's needed](https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/MetaTags.html) to enable the `apple-mobile-web-app-status-bar-style` meta tag and make it works. Do you know of any other alternative?